### PR TITLE
Fixing multiple issues with the layered repository service

### PR DESCRIPTION
**Pull request by tobiasqueck**
Contains:

- fix for #635 
- fix for #636
- fix to use `$tmp` when no package is provided
- enhancement of `examples/odata-cli` to also have a better way to debug issues like #636

### CODE REVIEW

**Review by testojs — DISMISSED** [review]
Looks great! Thanks Tobias!

**Review by testojs — DISMISSED** [review]

File: examples/odata-cli/README.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Example: CLI using Axios OData
 
-This project contains scripts to fetch OData metadata and annotations from using the `@sap-ux/axios-extension` and `@sap-ux/btp-utils` modules with real environments.
+This project contains scripts to fetch OData metadata and annotations from using the `@sap-ux/axios-extension` and `@sap-ux/btp-utils` modules with real environments. It also contains scripts showing how to work with the layer repository service and the ADT services.
```
Comment (ullasholla) [inline examples/odata-cli/README.adoc:3]:
```suggestion
This project contains scripts to fetch OData metadata and annotations by using the `@sap-ux/axios-extension` and `@sap-ux/btp-utils` modules with real environments. It also contains scripts showing how to work with the layered repository service and the ADT services.
```
Does sound better?

**Review by ullasholla — CHANGES_REQUESTED** [review]
Thanks @tobiasqueck, looks good. Had a minor note about the `README`: https://github.com/SAP/open-ux-tools/pull/637/files#r954428623

**Review by ullasholla — APPROVED** [review]
Thanks @tobiasqueck

**Pull request merged**